ui/perfherder/compare/CompareTableView.jsx

```diff
@@ -274,9 +277,15 @@ export default class CompareTableView extends React.Component {
                 <Row>
                   <Col sm="12" className="text-center pb-1">
                     <h1>
-                      {hasSubtests
-                        ? `${title} subtest summary`
-                        : 'Perfherder Compare Revisions'}
+                      <ComparePageLabel
+                        label={
+                          hasSubtests
+                            ? `${title} subtest summary`
+                            : 'Perfherder Compare Revisions'
+                        }
+                        updateParams={updateParams}
+                        pageLabelQueryParam={getUrlParam('pageLabel')}
```

Comment:
I don't think you need `getUrlParam('pageLabel')`; you can unpack `pageLabel` from `this.props.validated`.

Reply:
OK